docs/Feature.md

```diff
@@ -0,0 +1,54 @@
+# Using GordianSigner
+
+*GordianSigner* allows you to: sign a PSBT, import & managing your accounts, fill & export account map and save contacts.
+This instructions show how to use these basic features.
+
+## Singing a PSBT
+
+<img src="images/sign_psbt.png" alt="" width="250"/> <img src="images/sign_psbt_import.png" alt="" width="250"/>
+
+You can import a PSBT to sign by pasting from clipboard, import from file or scan UR code.
```

Comment:
I should be `importing` and `scanning` as well.

Reply:
There are lots of ways to import a PSBT here, so I think just showing the result after importing/scanning & describe in the description is enough. Anyway, these steps are very common like scanning or browsing file.

Reply:
@hvthhien I mean the spelling mistake.

Reply:
okay, I see